Samplero/Samplero/Screen/Camera/CameraViewController.swift

```diff
@@ -0,0 +1,260 @@
+//
+//  CameraViewController.swift
+//  Samplero
+//
+//  Created by JiwKang on 2022/10/09.
+//
+
+import UIKit
+import AVFoundation
+
+import SnapKit
+
+class CameraViewController: BaseViewController {
+    
+    // MARK: - Properties
+    
+    // Capture Session
+    var session: AVCaptureSession?
+    // Photo Output
+    let output = AVCapturePhotoOutput()
+    // Video Preview
+    let previewLayer = AVCaptureVideoPreviewLayer()
+    
+    // Shutter Button
+    private let shutterButton: UIButton = {
+        let buttonSize: CGFloat = UIScreen.main.bounds.height / 14.07
+        let boarderSize: CGFloat = buttonSize + buttonSize / 5
+        
```

Comment:
p5. 우선 저의 nameSpace에 대한 개인적인 의견임을 밝힙니다 ㅎㅎ. 우기방식도 좋구 아니면 class CameraViewController 바깥 위쪽에 
```suggestion
private enum Size {
  static let buttonSize = UIScreen.main.bounds.height / 14.07
  static let boarderSize: CGFloat = buttonSize + buttonSize / 5
}
        
```
작성하는 방법도 있어요!. 이 파일에서만 사용될 것이기 때문에 private을 사용하였고 struct가 아니라 enum을 사용하는 이유는 struct의 경우 다른개발자가 실수로라도 인스턴스를 만들 수 있기때문에 방지하고자..!

Reply:
너무 다양한 사이즈가 나올 것 같아서 저 방식으로 진행하였습니다. 나중에 논의해보시죠!